.gitlab/maintenance_jobs/kitchen.yml

```diff
@@ -38,8 +38,6 @@ periodic_kitchen_cleanup_azure:
     - RESOURCE_GROUP_PREFIX=kitchen-win python3 /deploy_scripts/cleanup_azure.py
     - RESOURCE_GROUP_PREFIX=kitchen-security-agent python3 /deploy_scripts/cleanup_azure.py
     - RESOURCE_GROUP_PREFIX=kitchen-system-probe python3 /deploy_scripts/cleanup_azure.py
-    - RESOURCE_GROUP_PREFIX=kitchen-process-collection python3 /deploy_scripts/cleanup_azure.py
-    - RESOURCE_GROUP_PREFIX=kitchen-container-collection python3 /deploy_scripts/cleanup_azure.py
```

Comment:
I'd do so in a separate step, a few days later, otherwise the resources created just before the merge of this PR may never get cleaned up.

Reply:
Oh I was not aware of this. Let me revert and I'll open another PR in a couple of days. Thank you